pontoon/sync/vcs/models.py

```diff
@@ -459,10 +459,15 @@ def project(self):
         """Return parsed project configuration file."""
         path = os.path.join(
             self.vcs_project.db_project.source_repository.checkout_path,
-            self.configuration_file
+            self.configuration_file,
         )
-        parser = TOMLParser()
-        return parser.parse(path)
+
+        # Use parent directory of the first locale directory as l10n_base
+        l10n_base = get_parent_directory(
+            self.vcs_project.locale_directory_paths.itervalues().next()
+        )
```

Comment:
Until [bug 1497851](https://bugzilla.mozilla.org/show_bug.cgi?id=1497851) is fixed.

Reply:
I'm hand-waving, but I think this is not right. `source_directory_path` might be closer, and is a less scary code path? Or, `self.vcs_project.db_project.source_repository.checkout_path`, which I think should be correct for non-gecko apps.